exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/metadata/MetadataAggregateHelper.java

```diff
@@ -117,16 +120,16 @@ private void createAggregatorInternal() {
       }
     }
 
-    for (SchemaPath excludedColumn : excludedColumns) {
-      if (excludedColumn.equals(SchemaPath.getSimplePath(columnNamesOptions.rowGroupStart()))
-          || excludedColumn.equals(SchemaPath.getSimplePath(columnNamesOptions.rowGroupLength()))) {
-        LogicalExpression lastModifiedTime = new FunctionCall("any_value",
+    for (SchemaPath nonSchemaColumn : context.metadataColumns()) {
```

Comment:
Rename to `metadataColumn` or `implicitColumn`?

Reply:
Sorry, missed it.